### PR TITLE
Create pylint.yml and apply clean code

### PR DESCRIPTION
**Pull request by nitro2**
Test Action workflow.
Verify python clean code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +169,7 @@ gameplay created!
 FINISH ALL TEST CASES SUCCESSFULLY
 ```
 
-Tips: you can reduce the DELAY_TIME for faster run
+Tips: you can reduce the delay_time for faster run
```
Comment (huybt22) [inline README.md:172]: Should be `TEST_THREAD_DELAY_TIME` in env file

File: json/text_template.json [hunk 2]
```diff
@@ -40,6 +40,14 @@
     "coin_tail": {
       "vi": "sấp",
       "en": "tail"
+    },
+    "turn_on": {
+      "vi": "bật",
+      "en": "on"
+    },
+    "turn_off": {
```
Comment (huybt22) [inline json/text_template.json:48]: should be capitalized?

**Comment by huybt22** [comment]
Is there a common auto linting format config file for devs?

File: json/text_template.json (hunk 2 above)
Comment (nitro2) [reply json/text_template.json:48]: No..

**Comment by nitro2** [comment]
No, we do not strictly force to use JSON format as it can make text hard to read in some cases. We will consider it if it really brings profit to do it.

**Comment by nitro2** [comment]
Merge to speed up other issues implementation. Should base on this PR final commit.

**Pull request merged**